src/main/java/edu/unc/lib/boxc/migration/cdm/options/DestinationMappingOptions.java

```diff
@@ -34,10 +34,15 @@ public class DestinationMappingOptions {
                     "The default will be used during SIP generation for objects if they do not have explicit mappings.",})
     private String fieldName;
 
-    @Option(names = { "-f", "--force"},
+    @Option(names = {"-f", "--force"},
             description = "Overwrite destination mapping if one already exists")
     private boolean force;
 
+    @Option(names = {"-ac", "--archival-collections"},
```

Comment:
Does picocli support multi-letter short options? I actually don't know, and I'm having a hard time telling from the documentation. I'm fine this if it works though and you've verified it, I guess one of the tests could be updated to reference it with the short form.
https://picocli.info/man/2.x/#_options_and_parameters